src/components/KanbanComponent/KanbanComponent.tsx

```diff
@@ -26,29 +13,125 @@ import { CardComponent } from '../CardComponent';
 import DEFINED_PRs from '../../res/PRS_DEFINED';
 import IN_PROGRESS_PRs from '../../res/PRS_INPROGRESS';
 import DONE_PRs from '../../res/PRS_DONE';
-import { PreviewComponent, QueryType } from '../PreviewComponent/PreviewComponent';
+import { PreviewComponent } from '../PreviewComponent/PreviewComponent';
 import { KanbanColumnHeader } from './KanbanColumnComponent/KanbanColumnHeaderComponent';
 import { KanbanTeamsComponent } from './KanbanTeamsComponent';
 
+import { githubAuthApiRef, useApi } from '@backstage/core-plugin-api';
+
+export type CommentType = {
+  author: {
+    login: string;
+  };
+  body: string;
+  createdAt: string;
+  updatedAt: string;
+};
+
+export type ReviewType = {
+  author: {
+    login: string;
+  };
+  body: string;
+  comments: {
+    nodes: CommentType[];
+  };
+  createdAt: string;
+  state: string;
+  updatedAt: string;
+};
+
+export type LabelType = {
+  color: string;
+  name: string;
+};
+
+export type PRType = {
+  author: {
+    login: string;
+  };
+  body: string;
+  createdAt: string;
+  number: number;
+  labels: {
+    nodes: LabelType[];
+  };
+  reviews: {
+    nodes: ReviewType[];
+  };
+  state: string;
+  title: string;
+  updatedAt: string;
+  url: string;
+};
+
+export type RepoType = {
+  repository: string;
+  data: PRType[];
+};
+
 export const KanbanComponent = () => {
-  const [query, setQuery] = useState<QueryType>();
+  const [query, setQuery] = useState<PRType>();
+  const [userRepos, setUserRepos] = useState<RepoType[]>();
+  const [userRepoNames, setUserRepoNames] = useState<String[]>();
```

Comment:
Use `string` with a lower case
```suggestion
  const [userRepoNames, setUserRepoNames] = useState<string[]>();
```